nucliadb/src/nucliadb/writer/resource/field.py

```diff
@@ -133,6 +133,8 @@ async def extract_fields(resource: ORMResource, toprocess: PushPayload):
                         await processing.convert_internal_cf_to_str(cf, storage)
                         for cf in message.content.attachments
                     ]
+                    if "attachments_fields" in parsed_message["content"]:
```

Comment:
why are you deleting this? Maybe this deserves a comment on the code

Reply:
Not defined on the push payload